### PR TITLE
Plot running times

### PR DESCRIPTION
**Pull request by oharboe**
I was hoping to plot detailed routing time as a function of area, but so far this script has been useful to generate various test-cases(segfaults or detailed routing errors), so I thought it would be useful to share.

### CODE REVIEW

**Comment by oharboe** [comment]
@maliberty The first commit is in master already, so this PR has only the utility script

File: flow/util/plot-running-times.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        # 3_3_place_gp                       5
+        # 3_4_resizer                        2
+        # 3_5_opendp                         3
+        # 4_1_cts                            5
+        # 4_2_cts_fillcell                   3
+        # 5_1_fastroute                      5
+        # 5_2_TritonRoute                  297
+        # 6_1_merge                          3
+        # 6_report                          67
+        pattern = r'^5_2_TritonRoute\s+(\d+)$'
```
Comment (maliberty) [inline flow/util/plot-running-times.py]: Now 5_2_route.log

Comment (oharboe) [reply flow/util/plot-running-times.py]: Fixed

File: flow/util/plot-running-times.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
+import subprocess
```
Comment (maliberty) [inline flow/util/plot-running-times.py]: This script is specific to mock array. It should therefore be in the design dir

**Comment by oharboe** [comment]
@maliberty Good to go.

**Review by maliberty — CHANGES_REQUESTED** [review]
Sorry I reviewed but forgot to hit submit.

Are all the non-plotting changes intended here?

**Comment by oharboe** [comment]
> Sorry I reviewed but forgot to hit submit.
> 
> Are all the non-plotting changes intended here?

Those are already in master.

**Review by maliberty — APPROVED** [review]

**Pull request merged**